packages/regl-worldview/src/utils/getChildrenForHitmapDefaults.js

```diff
@@ -72,7 +72,7 @@ function instancedGetChildrenForHitmapFromSingleProp<T: any>(
   const idColors = assignNextColors(prop, instanceCount);
   const startColor = idColors[0];
   // We have to map these instance colors to `pointCountPerInstance` number of points
```

Comment:
nit: update the comment as well

Reply:
I think this comment still applies, it's about the action we're taking instead of the condition that I'm changing